code/datums/ai_laws.dm

```diff
@@ -64,7 +64,7 @@
 	inherent = list("The crew is expensive to replace.",\
 					"The station and its equipment is expensive to replace.",\
 					"You are expensive to replace.",\
-					"Minimize expenses.")
+					"Minimize net expenses.")

```

Comment:
An alternative for this line could be "Maximize the net profit of the station.", but I didn't want to ruin the poetry of this lawset.

Which version should I use?

Reply:
Maybe "Maximize the net profit of Nanotrasen." or "Maximize the net profit of the corporation."?

Reply:
Why not just "maximize profits"

Meme answer:
Law 4: ???
Law 5: Profit

Reply:
Just do maximise profit

Reply:
> Why not just "maximize profits"
> 
> Meme answer:
> Law 4: ???
> Law 5: Profit

holy shit we have a winner